src/res/translation/translation_zh_CN.ts

```diff
@@ -0,0 +1,3072 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="zh_CN">
+<context>
+    <name>CAboutDlg</name>
+    <message>
+        <location filename="../../util.cpp" line="412"/>
+        <source>Qt cross-platform application framework</source>
+        <translation>Qt 跨平台应用程序框架</translation>
+    </message>
+    <message>
+        <location filename="../../util.cpp" line="417"/>
+        <source>Audio reverberation code by Perry R. Cook and Gary P. Scavone</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../util.cpp" line="421"/>
+        <source>Some pixmaps are from the</source>
+        <translation>部分图像资源来自</translation>
+    </message>
+    <message>
+        <location filename="../../util.cpp" line="389"/>
+        <source>This app enables musicians to perform real-time jam sessions over the internet.</source>
+        <translation>此应用使乐手们可以在线实时进行音乐合奏。</translation>
+    </message>
+    <message>
+        <location filename="../../util.cpp" line="392"/>
+        <source>There is a server which collects  the audio data from each client, mixes the audio data and sends the mix  back to each client.</source>
```

Comment:
Issue in English: there's a space too much.

Reply:
While working on the translation, I found there are also some other issues in the original translation strings, I will discuss them in separated issue or PR after this one get merged :)

Reply:
Yes. Thanks. Would be great.